server/lib/activities.js

```diff
@@ -29,7 +30,7 @@ module.exports = {
     // get group data
     if (activity.data.group) {
       groupName = activity.data.group.name;
-      publicUrl = activity.data.group.publicUrl;
+      publicUrl = `https://opencollective.com/${activity.data.group.slug}`;
```

Comment:
Why add the url manually? It's going to break in non-production places. publicUrl is meant to serve this link. Is it not working?